components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -194,7 +194,7 @@ const PrepopulatedEngine brave_search = MakeBravePrepopulatedEngine(
     "desktop",
 #endif
     "UTF-8",
-    "https://search.brave.com/api/suggest?q={searchTerms}",
+    "https://search.brave.com/api/suggest?q={searchTerms}&rich=true",
```

Comment:
I think `kBraveCurrentDataVersion` should be updated.

Reply:
Nice find! Thanks